services/brig/test/integration/API/User/Account.hs

```diff
@@ -1257,6 +1258,56 @@ testDeleteWithRemotes opts brig = do
           (eitherDecode . cs) (F.body r)
         Nothing -> Left "No request"
 
+testDeleteWithRemotesAndFailedNotifications :: Opt.Opts -> Brig -> Cannon -> Http ()
+testDeleteWithRemotesAndFailedNotifications opts brig cannon = do
+  alice <- randomUser brig
+  alex <- randomUser brig
+  let localDomain = qDomain (userQualifiedId alice)
+
+  let bDomain = Domain "b.example.com"
+      cDomain = Domain "c.example.com"
+  bob <- Qualified <$> randomId <*> pure bDomain
+  carl <- Qualified <$> randomId <*> pure cDomain
+
+  postConnection brig (userId alice) (userId alex) !!! const 201 === statusCode
+  putConnection brig (userId alex) (userId alice) Accepted !!! const 200 === statusCode
+  sendConnectionAction brig opts (userId alice) bob (Just FedBrig.RemoteConnect) Accepted
+  sendConnectionAction brig opts (userId alice) carl (Just FedBrig.RemoteConnect) Accepted
+
+  let fedMockResponse req =
+        if Domain (F.domain req) == bDomain
+          then F.OutwardResponseError (F.OutwardError F.ConnectionRefused "mocked connection problem with b domain")
+          else OutwardResponseBody (cs $ Aeson.encode EmptyResponse)
+
+  let galleyHandler :: ReceivedRequest -> MockT IO Wai.Response
+      galleyHandler (ReceivedRequest requestMethod requestPath _requestBody) =
+        case (requestMethod, requestPath) of
+          (_methodDelete, ["i", "user"]) -> do
+            let response = Wai.responseLBS Http.status200 [(Http.hContentType, "application/json")] (cs $ Aeson.encode EmptyResponse)
+            pure response
+          _ -> error "not mocked"
```

Comment:
There's no need to pattern match on `requestMethod` as it is vacuous, right?

Reply:
Yes, you're right. I changed this to a proper pattern match in https://github.com/wireapp/wire-server/pull/1912/commits/d92e7fa976992f97f17733d0147c2fecd0d7e88b